.github/workflows/release.yml

```diff
@@ -1,5 +1,5 @@
-# Copyright (c) 2023 Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
-# Copyright (c) 2023 Contributors to the Eclipse Foundation
+# Copyright (c) 2021-2022 Copyright (c) 2021-2022 Robert Bosch Manufacturing Solutions GmbH
```

Comment:
Please use the newest copyright header (2023)

Reply:
incorporated the newest copyright header (2023)